ui/pages/swaps/awaiting-swap/view-on-ether-scan-link/view-on-ether-scan-link.js

```diff
@@ -1,50 +0,0 @@
-import React, { useContext } from 'react';
-import PropTypes from 'prop-types';
-import classnames from 'classnames';
-import { I18nContext } from '../../../../contexts/i18n';
-import { getURLHostName } from '../../../../helpers/utils/util';
-import { MetaMetricsContext } from '../../../../contexts/metametrics';
-import { EVENT } from '../../../../../shared/constants/metametrics';
-
-export default function ViewOnEtherScanLink({
-  txHash,
-  blockExplorerUrl,
-  isCustomBlockExplorerUrl,
-}) {
-  const t = useContext(I18nContext);
-  const trackEvent = useContext(MetaMetricsContext);
-
-  return (
-    <div
-      className={classnames('awaiting-swap__view-on-etherscan', {
-        'awaiting-swap__view-on-etherscan--visible': txHash,
-        'awaiting-swap__view-on-etherscan--invisible': !txHash,
-      })}
```

Comment:
Did we intentionally remove this functionality from the new `ViewOnBlockExplorerURL` component? If so why?

Reply:
Oh wait nvm the `ViewOnBlockExplorer` is only shown if `txHash` is truthy